src/taskgraph/main.py

```diff
@@ -88,10 +88,12 @@ def get_filtered_taskgraph(taskgraph, tasksregex, exclude_keys):
                     if regexprogram.match(dep):
                         filterededges.add((key, dep, depname))
 
-        taskgraph = TaskGraph(filteredtasks, Graph(set(filteredtasks), filterededges))  # type: ignore
+        taskgraph = TaskGraph(
+            filteredtasks, Graph(frozenset(filteredtasks), frozenset(filterededges))
+        )
 
     if exclude_keys:
-        for label, task in taskgraph.tasks.items():  # type: ignore
+        for label, task in taskgraph.tasks.items():
```

Comment:
Are you referring to `load_tasks` in `Kind` class in `generator.py`? I tried changing it, but it seems the tests might need to be re-written as well in which case this can be a new issue by itself. Unless I misunderstood what you meant. Let me know if I am missing something. I have left this unchanged for now.

Reply:
Oh, my apologies. I was talking about https://github.com/taskcluster/taskgraph/blob/f0d58f2f23d809ab27f1745d661ac27587da740c/src/taskgraph/taskgraph.py#L24, but I just noticed that it's already been fixed...I was looking at outdated code locally when doing part of this review :facepalm: